package.json

```diff
@@ -31,5 +31,6 @@
             "_comment": "This will remove husky from when we started migrating to use prettier",
             "pre-commit": "npm uninstall husky"
         }
-    }
+    },
+    "dependencies": {}
```

Comment:
Please revert this change in package.json

Reply:
Oops! npm or yarn (forget what I ran) must have automatically done this when re-running install and test before committing. This is reverted now.